src/components/AuthenticationMainText.css

```diff
@@ -0,0 +1,14 @@
+.mainTextContainer {
+    display: flex;
+    flex-direction: column;
+}
+
+.mainTextHeader {
+    font-size: 3.3em;
+    color:#00151A;
+}
+
+.mainTextBody {
+    font-size: 1.5em ;
+    color: #55666A;
+}
```

Comment:
Please, move this file into the styles folder as it is not a component

Reply:
As and the time I pushed, you had not merged. Didn't want to create another folder.